src/app/pages/profile/transaction-information/transaction-information.component.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { DebugElement } from '@angular/core';
+import { HttpClientModule } from '@angular/common/http';
+import { of } from 'rxjs';
+import { By } from '@angular/platform-browser';
+
+import { AppModule } from './../../../app.module';
+import { resetSpies, transactionServiceSpy } from './../../../helpers/spies';
+import { TransactionInformationComponent } from './transaction-information.component';
+import { TransactionInformationService } from 'src/app/services/profile/transaction-information/transaction-information.service';
+import { EmptyTransactionComponent } from './empty-transaction/empty-transaction.component';
+
+describe('TransactionInformationComponent', () => {
+  let component: TransactionInformationComponent;
+  let fixture: ComponentFixture<TransactionInformationComponent>;
+  let debugElement: DebugElement;
```

Comment:
we can use `const` here since `debugElement` is not re-assigned

Reply:
I worked on that @Tevinthuku